src/query_optimizer/statement2plantree.py

```diff
@@ -0,0 +1,48 @@
+from src.query_parser.select_statement import SelectStatement
+from query_planner.logical_select_plan import LogicalSelectPlan
+from query_planner.logical_inner_join_plan import LogicalInnerJoinPlan
+from query_planner.logical_projection_plan import LogicalProjectionPlan
+from query_planner.logical_select_plan import LogicalSelectPlan
+from query_planner.video_table_plan import VideoTablePlan
+from loaders.video_loader import SimpleVideoLoader
+from models.catalog.video_info import VideoMetaInfo
+from models.catalog.properties import VideoFormat
+
+
+# class that will handle converting a statement to a plan tree
+class Statement2Plantree:
+    def __init__(self):
+        pass
+
+    # Function that converts the statement list to a plan tree
```

Comment:
Comments are not as per our standards. And why is there a class for just one static function? We need to keep some sort of framework where other modules just call a single function with evaStatementList and then it is the job of this module to figure out which type of statement it is and call the corresponding converter.